Automattic/woocommerce-payments
# Add Account Balances foundation component

Resolves #5720

#### Changes proposed in this Pull Request









This PR adds the foundation for a new "Account Balances" card to the **Payments → Overview** admin screen.
* A new `<AccountBalances />` component.
* A `style.scss` and test suite within the component's `test` subdir.

```
client/components/account-balances
├── index.tsx
├── large-child-component
│   └── test
├── style.scss
└── test
    ├── __snapshots__
    │   └── index.test.tsx.snap
    └── index.test.tsx
```

<img width="690" alt="image" src="https://user-images.githubusercontent.com/3147296/224617089-c27dd6e0-d8ea-4607-b401-c3f14b02e7f3.png">

When the feature flag `_wcpay_feature_simplify_deposits_ui` from #5690 is enabled, this new `<AccountBalances />` card will replace the current [`<DepositsInformation />`](https://github.com/Automattic/woocommerce-payments/blob/3739907449a701495ca013618c58d6dbab7bf02b/client/overview/index.js#L142) card.

#### Testing instructions





* Run `npm run build:client`.
* View the **Payments → Overview** screen.
* The `_wcpay_feature_simplify_deposits_ui` feature flag is disabled by default.
* You should see the current `<DepositsInformation />` card, **not** the `<AccountBalances />` card.

   <img width="686" alt="image" src="https://user-images.githubusercontent.com/3147296/224617418-444841da-9051-4c9f-92bf-73a53bbfb9a3.png">

* Enable the `_wcpay_feature_simplify_deposits_ui` feature flag.  
To enable, run `update_option( '_wcpay_feature_simplify_deposits_ui', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/).
* You should see the new `<AccountBalances />` card and **not** the `<DepositsInformation />` card.

   <img width="690" alt="image" src="https://user-images.githubusercontent.com/3147296/224617089-c27dd6e0-d8ea-4607-b401-c3f14b02e7f3.png">

-------------------


**Post merge**
